pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -171,6 +171,15 @@ func (p *provider) DeleteNodes(n []nodes.Node) error {
 	return deleteVolumes(nodeVolumes)
 }
 
+// getHostIPOrDefault defaults HostIP to localhost if is not set 
```

Comment:
The test failure appears to be coming from here. I had to look locally to see you have a space at the end of this line that gets removed by the `make update` target. So you should be able to just run that and check in the updated changes.

Reply:
Thank you very much!